doc/man5/config.pod

```diff
@@ -480,6 +481,50 @@ This sets the property query used when fetching the randomness source.
 
 =back
 
+=head2 ASN.1 String-Table Configuration
+
+The name B<stbl_section> in the initialization section names the section
+containing overrides for the build-in ASN.1 string-table size limits
+(defined in C<tbl_standard.h>). Within this section, each entry is a
+name/value pair where the B<name> is an ASN.1 object short or long name,
+and the B<value> is a comma-separated list of parameters in the form
+C<min:<n>>, C<max:<n>>, C<mask:<mask>>, or C<flags:<flags>>.
+
+=over 4
+
+=item C<min> and C<max>
+
+Specify the minimum and maximum character counts, respectively.
+
+=item C<mask>
+
+Passed through L<ASN1_str2mask(3)>; defines which ASN.1 string encodings are
```

Comment:
Isn't `ASN1_str2mask` an irrelevant implementation detail? It's not documented anyway.

Reply:
good point, I'll edit the sentence.

Reply:
actually no, it's a public function.  just not documented so the link will fail.  I removed it anyway.